documentation/List-of-Action-plugins.md

```diff
@@ -658,6 +659,26 @@ Can supply *either* `applicationId` or `applicationSerial`, although this can be
 
 Note: There is a database trigger/postgres listener to automatically delete files when their database record is deleted, so we only need to delete the records, not the files themselves.
 
+### Aliasing existing template actions
+
+Finally, we have this "special" action which allows us to create an alias to an existing template action. The use for this is if you have configured a complex action with a lot of parameters or queries, and you wish to run it with slight differences for a document preview (for example), you can just set an alias to the original action instead of configuring and maintaining the same thing twice within the template (LINK TO PREVIEW DOCUMENTATION WHEN ITS DONE)
```

Comment:
Can we add a quick term glossary, that `alias` means this action and `aliased` means the action it is referring too, sorry I got a little bit confused about the termss

Reply:
In my latest doc update (5b22f6fc) I've tried to keep to "alias" for the new action, and "original action" for the one it references. This okay?

Reply:
thanks